packages/yew-functional-macro/tests/function_attr/generic-lifetime-fail.rs

```diff
@@ -1,18 +1,18 @@
-use yew::prelude::*;
-use yew_functional::function_component;
-
-#[derive(Clone, Properties, PartialEq)]
-struct Props {
-    a: usize,
-}
-
-#[function_component(Comp)]
-const fn comp<P: Properties>(props: &P) -> Html {
-    html! {
-        <p>
-            { props.a }
-        </p>
-    }
-}
-
-fn main() {}
+use yew::prelude::*;
+use yew_functional::function_component;
+
+#[derive(Clone, Properties, PartialEq)]
+struct Props {
+    a: usize,
+}
+
+#[function_component(Comp)]
+fn comp<'a>(props: &Props) -> Html {

```

Comment:
This isn't an issue right now, but let's actually use the lifetime parameter just to be sure.

```suggestion
fn comp<'a>(props: &'a Props) -> Html {

```